monkey/monkey_island/cc/resources/attack_config.py

```diff
@@ -0,0 +1,30 @@
+import flask_restful
+import json
+from flask import jsonify, request
+
+from cc.auth import jwt_required
+from cc.services.attack.attack_config import *
+
+__author__ = "VakarisZ"
+
+
+class AttackConfiguration(flask_restful.Resource):
+    @jwt_required()
+    def get(self):
+        return jsonify(configuration=get_config()['properties'])
+
+    @jwt_required()
+    def post(self):
+        """
+        Based on request content this endpoint either resets ATT&CK configuration or updates it.
+        :return: Technique types dict with techniques on reset and nothing on update
+        """
+        config_json = json.loads(request.data)
+        if 'reset_attack_matrix' in config_json:
```

Comment:
Does resetting this reset the normal config as well?

Reply:
No